client/app/error/error.routing.js

```diff
@@ -0,0 +1,19 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.error', {
+    url: '/error',
+    params: {
+      detail: null,
+    },
+    views: {
+      'app@': {
```

Comment:
Is there a reason to not specify the state where the `app` view is located ?

Reply:
The `app` ui-view doesn't seem associated with any states